UWP/App.cpp

```diff
@@ -277,7 +295,8 @@ void App::OnVisibilityChanged(CoreWindow^ sender, VisibilityChangedEventArgs^ ar
 		// MainScreen::OnExit and even App::OnWindowClosed
 		// doesn't seem to be called when closing the window
 		// Try to save the config here
-		g_Config.Save("App::OnVisibilityChanged");
+		// OnSuspending should now do the job
+		//g_Config.Save("App::OnVisibilityChanged");
```

Comment:
If this isn't needed, I recommend removing the entire if.  Git will remember.

-[Unknown]